.env.development

```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_HOST=http://localhost
+REACT_APP_TOKEN_TIMEOUT=43200000
```

Comment:
The token life in the backend is 43200, I don't know if this is the same.

Reply:
It's 12 hours and written in millisecond scope because I don't want to add x1000 to use it. :)